packages/mlgmpidl/mlgmpidl.1.2.11/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+authors: ["Bertrand Jeannet" "Nicolas Berthier"]
+maintainer: "Nicolas Berthier <m@nberth.space>"
+dev-repo: "git+https://github.com/nberth/mlgmpidl.git"
+bug-reports: "https://github.com/nberth/mlgmpidl/issues"
+homepage: "https://www.inrialpes.fr/pop-art/people/bjeannet/mlxxxidl-forge/mlgmpidl/"
+license: "LGPL-2.1 with static linking exception"
+build: [
+  ["sh" "./configure" "--absolute-dylibs" { os = "darwin" } ]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: [
```

Comment:
This is not needed since opam 2.0

Reply:
Thanks for notice. Does this also work fine if sandboxing is disabled?

Reply:
It should but better ask more knowledgeable people: ping @kit-ty-kate @rjbou

Reply:
I'm just assuming it would if the sand-boxing mechanism is not used to track changes on the file system as well.